pettingzoo/tests/api_test.py

```diff
@@ -180,17 +213,20 @@ def progress_report(msg):
     assert isinstance(env, pettingzoo.AECEnv), "Env must be an instance of pettingzoo.AECEnv"
 
     # do this before reset
-    observation = env.reset(observe=False)
-    assert observation is None, "reset(observe=False) must not return anything"
+    env.reset()
     assert not any(env.dones.values()), "dones must all be False after reset"
 
     assert isinstance(env.num_agents, int), "num_agents must be an integer"
     assert env.num_agents != 0, "Your environment should have a nonzero number of agents"
     assert env.num_agents > 0, "Your environment can't have a negative number of agents"
```

Comment:
```suggestion
    assert env.num_agents > 0, "Your environment should have a nonnegative number of agents"
```

Changed for consistency

Reply:
"An environment"?

Reply:
Fixed to 
`An environment should have a positive number of agents`